django_extensions/management/modelviz.py

```diff
@@ -119,9 +121,12 @@ def generate_graph_data(self):
                             relation['needs_node'] = False
 
     def get_graph_data(self, as_json=False):
-        now = datetime.datetime.now()
+        if self.deterministic:
+            date = datetime.datetime(2000, 1, 1, 0, 0)
```

Comment:
Would it make more sense to use Epoch Time (`January 1, 1970`) here ?

`date = datetime.datetime.fromtimestamp(0)`

Reply:
Agreed. Updated accordingly.